### DISCUSSION TITLE
Jina Install version

### DISCUSSION THREAD

**Discussion by poezie** [davejpeters/fabric#1] [Q&A]
Sorry I am new to this , how do I install the Jina version of Fabric ? I am not sure how to merge this with the original fabric solution

**Comment by davejpeters** [comment]
> Sorry I am new to this , how do I install the Jina version of Fabric ? I am not sure how to merge this with the original fabric solution 

The Jina tool is a separate tool from fabric, a helper of sorts, like the old YouTube helper (the new one is integrated into fabric). You can take the code from this repository under the "jina" folder and compile it with 'go build' or run it with 'go run'
